aws-lc-rs/src/agreement.rs

```diff
@@ -137,14 +151,18 @@ pub static X25519: Algorithm = Algorithm {
     id: AlgorithmID::X25519,
 };
 const X25519_PRIVATE_KEY_LEN: usize = aws_lc::X25519_PRIVATE_KEY_LEN as usize;
+#[cfg(test)]
 const ECDH_P256_PRIVATE_KEY_LEN: usize = 32;
+#[cfg(test)]
 const ECDH_P384_PRIVATE_KEY_LEN: usize = 48;
+const ECDH_P521_PRIVATE_KEY_LEN: usize = 66;
```

Comment:
I don't even know what `#[cfg(test)]` means but should we add it above the P521 line as well? :)

Reply:
interesting, thanks for explaining